scalardb-sample/database.properties

```diff
@@ -1,4 +1,29 @@
-scalar.db.storage=cassandra
-scalar.db.contact_points=localhost
-scalar.db.username=cassandra
-scalar.db.password=cassandra
+# For MySQL
+# scalar.db.storage=jdbc
+# scalar.db.contact_points=jdbc:mysql://localhost:3306/
+# scalar.db.username=root
+# scalar.db.password=mysql
+
+# For PostgreSQL.
+# scalar.db.storage=jdbc
+# scalar.db.contact_points=jdbc:postgresql://localhost:5432/
+# scalar.db.username=postgres
+# scalar.db.password=postgres
+
+# For DynamoDB
+# scalar.db.storage=dynamo
+# scalar.db.contact_points=dummy
+# scalar.db.username=dummy
+# scalar.db.password=dummy
```

Comment:
The word `dummy` has been deemed offensive, so we should avoid it. (Ref: [List of disability-related terms with negative connotations](https://en.wikipedia.org/wiki/List_of_disability-related_terms_with_negative_connotations))

How about if we replace it with `sample` instead?

Reply:
Thank you! I've revised it in 692b0d42cd60219488f03b3a544afd2667fe14dc.